aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/AwsMetricAttributeGenerator.java

```diff
@@ -94,14 +134,34 @@ private static void setService(Resource resource, SpanData span, AttributesBuild
    * name.
    */
```

Comment:
Please update comment, behaviour has substantially changed. Please review other comments in this code and update accordingly.